### DISCUSSION TITLE
upload_dsyms with app_store_connect_api_key

### DISCUSSION THREAD

**Discussion by NikKovIos** [fastlane/fastlane#18012] [Q&A]
I'm using [apiKey](https://docs.fastlane.tools/actions/app_store_connect_api_key/) but when making fastlane [upload_dsyms](https://docs.fastlane.tools/actions/upload_symbols_to_crashlytics/) it still request 2F Authorization. Need it to use API key if can.

**Comment by dinsen** [comment]
Hi @NikKovIos 
There is no endpoint on the App Store Connect API, that can download the dsyms. 
Thats why you get prompt with 2FA.
